docs/reference/commandline/node_ls.md

```diff
@@ -66,31 +68,55 @@ ID                           HOSTNAME        STATUS  AVAILABILITY  MANAGER STATU
 e216jshn25ckzbvmwlnh5jr3g *  swarm-manager1  Ready   Active        Leader
 ```
 
-### id
+#### ID
 
 The `id` filter matches all or part of a node's id.
 
 ```bash
 $ docker node ls -f id=1
-
 ID                         HOSTNAME       STATUS  AVAILABILITY  MANAGER STATUS
 1bcef6utixb0l0ca7gxuivsj0  swarm-worker2  Ready   Active
 ```
 
-#### label
+#### Label
 
-The `label` filter matches tasks based on the presence of a `label` alone or a `label` and a
-value.
+The `label` filter matches nodes based on the presence of a `label` alone or a `label` and a
+value. Label filtering in `docker node ls` now looks at engine labels rather than node labels. 
```

Comment:
Maybe we should simplify, because this sentence, and instead add an example that shows a filter on (e.g.) `label=somelabel=somevalue`;

```Markdown
The `label` filter matches nodes based on engine labels. Node labels are currently not used for filtering.
```

@vdemeester wdyt?